common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
You have to change the mesh so it wraps the waist.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120006130-a31ffd80-bfe9-11eb-913c-ad5c9a47b569.png)

</details>